addons/binding/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlThermostatHandler.java

```diff
@@ -58,7 +62,7 @@ public NikoHomeControlThermostatHandler(Thing thing) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        Integer thermostatId = ((Number) this.getConfig().get(CONFIG_THERMOSTAT_ID)).intValue();
+        String thermostatId = (String) this.getConfig().get(CONFIG_THERMOSTAT_ID);
```

Comment:
Configuration value de-marshaling is done on every handle command call, but should be done in the init method.
If multiple configuration values are involved, the configuration holder pattern (`MyConfigClass c = getConfig().as(MyConfigClass.class)`) should be used.